src/Soneta.Sdk/Sdk/Sdk.targets

```diff
@@ -33,20 +33,18 @@
     <Error Text="Soneta.Generator.exe is missing." Condition=" '@(SonetaGeneratorExe)' == '' " />
     <ItemGroup>
       <BusinessXmls Include="@(None)" Condition="$([System.String]::new('%(Filename)%(Extension)').EndsWith('business.xml', StringComparison.InvariantCultureIgnoreCase ))"/>
+      <ConfigXmls Include="@(EmbeddedResource)" Condition="$([System.String]::new('%(Filename)%(Extension)').EndsWith('config.xml', StringComparison.InvariantCultureIgnoreCase ))"/>
       <WorkingXmls Include="@(BusinessXmls)"/>
-      <WorkingXmls Include="@(EmbeddedResource)" Condition="$([System.String]::new('%(Filename)%(Extension)').EndsWith('config.xml', StringComparison.InvariantCultureIgnoreCase ))"/>
+      <WorkingXmls Include="@(ConfigXmls)"/>
     </ItemGroup>
   </Target>
 
   <Target Name="ExecuteSonetaGenerator" BeforeTargets="CoreCompile" DependsOnTargets="PrepareSonetaGeneratorAssets" 
           Condition=" ( '$(RunSonetaGenerator)' == '' OR '$(RunSonetaGenerator)' != 'false' ) AND $( [System.IO.Directory]::GetFiles( '$(ProjectDir)' , '*.business.xml', SearchOption.AllDirectories ).Length ) &gt; 0 "
-          Inputs="@(WorkingXmls)" Outputs="@(WorkingXmls->'%(RelativeDir)%(Filename).cs')">
+          Inputs="%(WorkingXmls.Identity)" Outputs="@(WorkingXmls->'%(RelativeDir)%(Filename).cs')">
 
-    <PropertyGroup>
-      <CurrentBusinessXml>%(WorkingXmls.Identity)</CurrentBusinessXml>
-    </PropertyGroup>
     <ItemGroup>
-      <SameAssemblyBusinessXmls Include="@(BusinessXmls)" Condition=" '%(BusinessXmls.Identity)' != '$(CurrentBusinessXml)' "/>
+      <SameAssemblyBusinessXmls Include="@(BusinessXmls)" Exclude="@(WorkingXmls->'%(Identity)'->Replace('.config.xml','.business.xml'))"/>
```

Comment:
Czemu ma służyć ten Replace( config.xml, business.xml ) ?

Reply:
Dlatego byłem za rozdzieleniem tych workflow, ale generalnie chodzi o to, że dla config.xml musimy wykluczyć jego business.xml